Rx.playground/Pages/Playground.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,22 @@
+/*:
+ > # IMPORTANT: To use **Rx.playground**:
+ 1. Open **Rx.xcworkspace**.
+ 1. Build the **RxSwift-macOS** scheme (**Product** → **Build**).
+ 1. Open **Rx** playground in the **Project navigator**.
+ 1. Show the Debug Area (**View** → **Debug Area** → **Show Debug Area**).
+ */
+import RxSwift
+/*:
+ # Try Yourself
+ 
+ It's time to play with Rx 🎉
+ */
+playgroundShouldContinueIndefinitely()
+
+example("Try yourself") {
+  let disposeBag = DisposeBag()
```

Comment:
In this case `just` will emit `completed` so disposeBag will be never used. How about just letting the function body empty? (with a single comment)

```swift
example("Try yourself") {
  // Observable.just("Hello, RxSwift!") ...
}
```